nautobot/utilities/tables.py

```diff
@@ -41,6 +41,22 @@ def __init__(self, *args, user=None, **kwargs):
         for cpf in ComputedField.objects.filter(content_type=obj_type):
             self.base_columns[f"cpf_{cpf.slug}"] = ComputedFieldColumn(cpf)
 
+        for relationship in Relationship.objects.filter(source_type=obj_type):
+            if not relationship.symmetric:
+                self.base_columns[f"cr_{relationship.slug}_src"] = RelationshipColumn(
+                    relationship, side=RelationshipSideChoices.SIDE_SOURCE
+                )
+            else:
+                self.base_columns[f"cr_{relationship.slug}_peer"] = RelationshipColumn(
+                    relationship, side=RelationshipSideChoices.SIDE_PEER
+                )
+
+        for relationship in Relationship.objects.filter(destination_type=obj_type):
+            if not relationship.symmetric:
+                self.base_columns[f"cr_{relationship.slug}_dst"] = RelationshipColumn(
+                    relationship, side=RelationshipSideChoices.SIDE_DESTINATION
+                )
```

Comment:
Just for future reference, perhaps:
```suggestion
                )
            # symmetric relationships are already handled above in the source_type case
```